server/__tests__/app.test.js

```diff
@@ -167,3 +167,80 @@ afterAll(() => {
         })
     })
   })
+
+  describe('6. PUSH /api/reviews/:review_id/comments', () =>{
+
+    test("status:201, returns a comment matching the parametric review_id", ()=>{
+        const newComment = {
+            username: "philippaclaire9",
+            body: "Loved the film, wicked!"
+        };
+        return request(app)
+        .post('/api/reviews/2/comments')
+        .send({newComment})
+        .expect(201)
+        .then((response)=>{
+            const comment = response.body.comment;
+            expect(comment.comment_id).toBe(7);
+            expect(comment.body).toBe( "Loved the film, wicked!");
+            expect(comment.review_id).toBe(2);
+            expect(comment.author).toBe("philippaclaire9");
+            expect(comment.votes).toBe(0);
+        })
+    })
```

Comment:
Could we add a test for what happens if unnecessary properties are sent over from the input? Just to check we ignore them correctly.